### PR TITLE
Configure dotenv

### LINKED ISSUE #10: Update back end to use environment variables
# Location
index.ts

# Motivation

# Requirements

# Design comps

### PR DESCRIPTION
**Pull request by ma-anwar**
# Description
Configure dotenv
Add .eslintlc
# Closes issue(s)
closes #10 
# How to test / repro

# Screenshots

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have tested this code
- [x] I have updated the Readme

# Other comments

### CODE REVIEW

File: backend/src/config/constants.ts [hunk 1]
```diff
@@ -1,8 +1,7 @@
-// @TODO: source appropriate constants from process.env
+// @TODO: Store client ideas and secrets in database.
 export const CLIENT_ID = "10000000000004";
 export const CLIENT_SECRET =
   "SULCuWVF4E1foAy5GKm7z7hQxWUcOhLZjoBvzSdBKzN3OyNW776oaT1h69O65ozk";
```
Comment (smknaqvi) [inline backend/src/config/constants.ts]: lets put these in the env for now

Comment (smknaqvi) [reply backend/src/config/constants.ts]: cuz they are different for u and i
ty

File: backend/src/index.ts [hunk 2]
```diff
@@ -37,5 +35,5 @@ passport.deserializeUser(
 app.use("/auth", authRoutes);
 
 app.listen(PORT, () => {
-  console.log(`Listening to requests on ${HOST}`);
+  console.log(`Listening to requests on ${HOST_ADDRESS}`);
```
Comment (smknaqvi) [inline backend/src/index.ts:38]: still put the port tho

Comment (smknaqvi) [reply backend/src/index.ts:38]:
```suggestion
  console.log(`Listening to requests on ${HOST_ADDRESS}:${PORT}`);
```

**Review by smknaqvi — APPROVED** [review]
lgtm, just add the secrete and client id for now into env

maybe call it like DEV_CLIENT_ID or w.e. cuz we probs gonna use a ton

File: backend/src/index.ts (hunk 2 above)
Comment (ma-anwar) [reply backend/src/index.ts:38]: Port is appended to host address

File: backend/src/config/constants.ts (hunk 1 above)
Comment (ma-anwar) [reply backend/src/config/constants.ts]: Beautiful poetry

**Pull request merged**